azure/deploy.sh

```diff
@@ -105,6 +105,23 @@ if [[ $OPENSHIFT_USER_PROVIDE == "false" ]]; then
   #   exit 22
   # fi
   # set -e
+  
+oc login -u $OCP_USERNAME -p $OCP_PASSWORD --server=https://api.${CLUSTER_NAME}.${BASE_DOMAIN}:6443
+log "==== Adding PID limits to worker nodes ===="
+cat <<EOF > azure-pidslimit.yaml
+apiVersion: machineconfiguration.openshift.io/v1
+kind: ContainerRuntimeConfig
+metadata:
+  name: azure-pidslimit
+spec:
+  containerRuntimeConfig:
+    pidsLimit: 231239
+  machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/worker: ''
+EOF
+
+oc create -f azure-pidslimit.yaml
```

Comment:
@vishwajitdandage-psl Since the file contents are same, I think we should keep the yaml file in git repo and use it from aws/deploy.sh and azure/deploy.sh. We can have a file at [REPO-ROOT]/templates/container-runtime-config.yml and use it in the script.

Reply:
Looks good !!